packages/hooks/src/useIsSsr.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react';
+
+export default function useIsSsr() {
```

Comment:
Can we add some tests? 

I wonder if this is what we're after: https://testing-library.com/docs/react-testing-library/api/#hydrate

Reply:
Unfortunately, it seems like [this wouldn't work](https://github.com/testing-library/react-testing-library/issues/1340). It also seems that SSR testing is a long-standing issue both for [hook](https://github.com/testing-library/react-testing-library/issues/1120) and [component](https://github.com/testing-library/react-testing-library/issues/561) testing, neither of which appear to be getting resolved anytime soon based on the issues.

Going off of some threaded workarounds I found, I've added a `renderHookServer` method to `@leafygreen/testing-lib`. It appears to solve the issue for SSR tests in a JSDOM env. Take a look when you get a chance, would love to get your thoughts!